jsettlers.logic/src/jsettlers/logic/buildings/Building.java

```diff
@@ -88,7 +82,7 @@
 	private ShortPoint2D pos;
 	private IBuildingsGrid grid;
 	private Player player;
-	private byte state = STATE_CREATED;
+	private EBuildingState state = EBuildingState.CREATED;
```

Comment:
We use those a lot for testing.

Some of the savegames and replay files are in the repository, I have some more locally. This makes developing the UI faster - simply load the savegame and see if all images look the way they should. No need to play through a whole game.

This is why every update that breaks savegames also breaks local development and our automated tests. Therefore we avoid it if it is not necessary.